src/modules/mim/aptinstall.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "AptInstallModel",
```

Comment:
>Apt [](http://example.com/codeflow?start=11&length=3)

Is this the Linux apt command name?  Can we replace this by a clear word, instead of the command name which may be cryptic to some?

Reply:
We suggest to change "Apt" with "Application". So names would become:

"ApplicationInstallModel"
"ApplicationInstall"

Reply:
We are thinking to use 'PackageManager' since we use Apt(Advanced Package Tool) to install/unistall/update packages.